### ISSUE TITLE
Incorrect labels read by ZoomText reader for radios on 'How do you want to sign in?' full page example

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#1396]
This was an issue we observed whilst visiting the Digital Accessibility Centre as they were auditing GOV.UK Frontend in May 2019.

The wrong label was being read out by ZoomText Reader when hovering each radio button on the 'How do you want to sign in?' full page example.

## Steps to reproduce

Visit the 'How do you want to sign in?' full page example using Internet Explorer 11 and ZoomText.

The example presents three options:

- Sign in with Government Gateway
- Sign in with GOV.UK Verify
- Sign in with a digital identity from another European country

Hover over each radio button in turn.

## Expected result
The corresponding label should be read out for each radio button.

## Actual result
ZoomText was reading 'Sign in with Government Gateway' for every radio button.

**Comment by 36degrees** [comment]
This turned out to be a bug in ZoomText, which we've reported:

## Overview

ZoomText Reader prioritises the name attribute instead of the id attribute when associating labels with inputs.

As per the HTML specification, the `for` attribute should reference the label by ID:
https://www.w3.org/TR/html52/sec-forms.html#element-attrdef-label-for

## Steps to reproduce

Given the following markup, with:
- multiple radios buttons - each with the same name, but different IDs, and;
- where one of the inputs has an ID which is the same as the name

```
<fieldset>
  <input id="foo" name="foo" type="radio" value="foo">
  <label for="foo">
    Foo
  </label>

  <input id="bar" name="foo" type="radio" value="bar">
  <label for="bar">
    Bar
  </label>
</fieldset>
```

With Reader enabled, with echo turned on for mouse hover, hover over each radio button in turn with ZoomText.

## Expected result:

- Hovering over the radio button associated with label 'Foo' should announce 'Foo'
- Hovering over the radio button associated with label 'Bar' should announce 'Bar'

## Actual result:

- Hovering over the radio button associated with label 'Foo' announces 'Foo'
- Hovering over the radio button associated with label 'Bar' also announces 'Foo'

Tested in ZoomText 2019 (2019.1904.80) and Internet Explorer 11 (11.0.14393.0) on Windows 10
